lib/validators/actions.js

```diff
@@ -180,7 +180,7 @@ export function runKilnjsValidators(schemas = {}, components = {}) {
 export function getSchemasWithValidationRules(schemas = {}) {
   const validationSchemas = {};
 
-  Object.keys(schemas).forEach(schemaName => schemas[schemaName].validation ? validationSchemas[schemaName] = _.cloneDeep(schemas[schemaName]) : null);
+  Object.keys(schemas).forEach(schemaName => _.get(schemas, `${schemaName}.validation`, []) ? validationSchemas[schemaName] = _.cloneDeep(schemas[schemaName]) : null);
```

Comment:
This validation is not needed since we are getting the schema name from from the schema, so `schemas[schemaName]` should always return something

Reply:
But if the schema is not returned in the kiln.js file this throws an error and breaks the validators.

Reply:
Makes sense. Maybe we should ignore that `kiln.js` or send a descriptive error.